### PR TITLE
Technology

### PR DESCRIPTION
**Pull request by josepplloo**
## Summary

- Tech Card implemented

## Screenshots

image

## Trello Card

https://trello.com/c/b335DqML/22-tecnolog%C3%ADa

### CODE REVIEW

File: maquetado/scss/tech.scss [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    content: '';
+    display: table;
+    height: 304px;
+    transform: translate(-11px,-265px);
+    width: 208px;
+    z-index: -1;
+  }
+
+
+  &__logo {
```
Comment (nmartinelli23) [inline maquetado/scss/tech.scss]: Why did you choose to use "__" in css classes?

File: maquetado/scss/tech.scss [hunk 2]
```diff
@@ -0,0 +1,47 @@
+.tech {
+  align-items: center;
+  background-color: $green-color;
+  border: none;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  height: 304px;
```
Comment (nmartinelli23) [inline maquetado/scss/tech.scss]: Set a max-height in 304px and height in 100%. Same with width.

File: maquetado/scss/tech.scss [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    transform: translate(-11px,-265px);
+    width: 208px;
+    z-index: -1;
+  }
+
+
+  &__logo {
+    & img {
+      height: $tech-logo-size;
+      padding: 0 0 16px 0;
```
Comment (nmartinelli23) [inline maquetado/scss/tech.scss]: No need of last "0".

File: maquetado/scss/tech.scss [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+      width: $tech-logo-size;
+    }
+  }
+
+  &__name {
+    padding: 0 0 16px 0;
```
Comment (nmartinelli23) [inline maquetado/scss/tech.scss]: No need of last "0".

File: maquetado/scss/tech.scss [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  }
+
+  &__release {
+    font-weight: bolder;
+    padding: 0 0 24px 0;
```
Comment (nmartinelli23) [inline maquetado/scss/tech.scss]: No need of last "0".

File: maquetado/scss/variables.scss [hunk 6]
```diff
@@ -6,6 +6,7 @@ $black: #000;
 $blue: #00ADEE;
 $green-color: #BED23C;
 $grey-color: #C4C4C4;
+$white: #fff;
```
Comment (nmartinelli23) [inline maquetado/scss/variables.scss]: Use uppercase, please.

File: maquetado/index.html [hunk 7]
```diff
@@ -10,5 +10,21 @@
 <body>
   <h1 class="title">Front-End Tools</h1>
   <h1 class="title">Back-End Tools</h1>
+  <article class="tech">
```
Comment (henryzarza) [inline maquetado/index.html]: Change this tag (`div`)

File: maquetado/index.html [hunk 8]
```diff
@@ -10,5 +10,21 @@
...
+    <div class="tech-logo">
```
Comment (henryzarza) [inline maquetado/index.html]: Why this div. I think that isn't necessary

File: maquetado/index.html [hunk 9]
```diff
@@ -10,5 +10,21 @@
...
+      <img src="./assets/react_icon.svg" alt="react icon">
+    </div>
+    <div class="tech-name">ReactJS</div>
```
Comment (henryzarza) [inline maquetado/index.html]: Change this tag (`h3`) for keeping the correct syntax and structure in the HTML

File: maquetado/index.html [hunk 10]
```diff
@@ -10,5 +10,21 @@
...
+    <div class="tech-release">
+      <p>
```
Comment (henryzarza) [inline maquetado/index.html]: Delete this `p`

File: maquetado/index.html [hunk 11]
```diff
@@ -10,5 +10,21 @@
 <body>
   <h1 class="title">Front-End Tools</h1>
   <h1 class="title">Back-End Tools</h1>
+  <article class="tech">
+    <div class="tech-logo">
+      <img src="./assets/react_icon.svg" alt="react icon">
+    </div>
+    <div class="tech-name">ReactJS</div>
+    <div class="tech-release">
```
Comment (henryzarza) [inline maquetado/index.html]: Change this tag (`h6`)

File: maquetado/index.html [hunk 12]
```diff
@@ -10,5 +10,21 @@
...
+      <p>
+        Release: 2013
+      </p>
+    </div>
+    <div class="tech-description">
+      <p>
```
Comment (henryzarza) [inline maquetado/index.html]: Delete this

File: maquetado/index.html [hunk 13]
```diff
@@ -10,5 +10,21 @@
...
+    <div class="tech-logo">
+      <img src="./assets/react_icon.svg" alt="react icon">
+    </div>
+    <div class="tech-name">ReactJS</div>
+    <div class="tech-release">
+      <p>
+        Release: 2013
+      </p>
+    </div>
+    <div class="tech-description">
```
Comment (henryzarza) [inline maquetado/index.html]:
```suggestion
    <p class="tech-description">
```

File: maquetado/scss/application.scss [hunk 14]
```diff
@@ -1,3 +1,4 @@
 @import './equalizer';
 @import './variables';
 @import './titles';
+@import './tech.scss';
```
Comment (henryzarza) [inline maquetado/scss/application.scss]:
```suggestion
@import './tech';
```

File: maquetado/scss/tech.scss [hunk 15]
```diff
@@ -0,0 +1,49 @@
+.tech {
+  align-items: center;
+  background-color: $green-color;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: Delete the `-color` isn't necessary
```suggestion
  background-color: $green;
```

File: maquetado/scss/tech.scss [hunk 16]
```diff
@@ -0,0 +1,49 @@
...
+  border: none;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: Delete this isn't necessary

File: maquetado/scss/tech.scss [hunk 17]
```diff
@@ -0,0 +1,49 @@
...
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: Delete this

File: maquetado/scss/tech.scss [hunk 18]
```diff
@@ -0,0 +1,49 @@
...
+  justify-items: center;
+  margin: 16px;
+  max-height: 304px;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: Delete this

File: maquetado/scss/tech.scss [hunk 19]
```diff
@@ -0,0 +1,49 @@
...
+  max-width: 208px;
+  padding: 16px;
+  width: 100%;
+
+  &::after {
+    background-color: $white;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: Delete this property

File: maquetado/scss/tech.scss [hunk 20]
```diff
@@ -0,0 +1,49 @@
...
+    border: 2px solid $green-color;
+    border-radius: 8px;
+    content: '';
+    display: table;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: Delete this

File: maquetado/scss/tech.scss [hunk 21]
```diff
@@ -0,0 +1,49 @@
...
+    height: 304px;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]:
```suggestion
    height: 100%;
```

File: maquetado/scss/tech.scss [hunk 22]
```diff
@@ -0,0 +1,49 @@
...
+    transform: translate(-11px,-265px);
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: I think this values are exact. Change this for relative units

File: maquetado/scss/tech.scss [hunk 23]
```diff
@@ -0,0 +1,49 @@
...
+    width: 208px;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]:
```suggestion
    width: 100%;
```

File: maquetado/scss/tech.scss [hunk 24]
```diff
@@ -0,0 +1,49 @@
...
+    z-index: -1;
+  }
+
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: Delete this empty line

File: maquetado/scss/tech.scss [hunk 25]
```diff
@@ -0,0 +1,49 @@
...
+
+  &-logo {
+    & img {
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]: I think that isn't necessary this nesting

File: maquetado/scss/tech.scss [hunk 26]
```diff
@@ -0,0 +1,49 @@
...
+      height: $tech-logo-size;
+      padding: 0 0 16px;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]:
```suggestion
      margin-bottom: 16px;
```

File: maquetado/scss/tech.scss [hunk 27]
```diff
@@ -0,0 +1,49 @@
...
+      width: $tech-logo-size;
+    }
+  }
+
+  &-name {
+    padding: 0 0 16px;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]:
```suggestion
    margin-bottom: 16px;
```

File: maquetado/scss/tech.scss [hunk 28]
```diff
@@ -0,0 +1,49 @@
...
+  }
+
+  &-release {
+    font-weight: bolder;
+    padding: 0 0 24px;
```
Comment (henryzarza) [inline maquetado/scss/tech.scss]:
```suggestion
    margin-bottom: 24px;
```

**Pull request merged**